src/app/Components/app.jsx

```diff
@@ -16,8 +16,16 @@ var App = React.createClass({
   createLock: function() {
     this.lock = new Auth0Lock(this.props.clientId, this.props.domain);
     this.lock.show({
+      callbackURL: 'http://localhost:3000/login',
       connections: ['facebook'],
       icon: '../../src/client/img/gyftee-icon.png',
+      popup: true
```

Comment:
the login popup lingers on the /friends route after successful login
TODO: fix so popup only shows once